pyproject.toml

```diff
@@ -4,26 +4,31 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "python-pkcs11"
-version = "0.7.0"
+version = "0.8.0"
```

Comment:
Please do not change the version manually, it will be changed automatically by the release process.